control-plane/csi-driver/controller/src/controller.rs

```diff
@@ -304,11 +304,9 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
     ) -> Result<tonic::Response<DeleteVolumeResponse>, tonic::Status> {
         let args = request.into_inner();
         tracing::trace!(volume.uuid = %args.volume_id, request = ?args);
-
```

Comment:
any reason to remove these lines?

Reply:
I had added logger to see what we receive as volume id in DeleteVolumeRequest. I might have removed extra line while removing that debug! logger

Reply:
A logger? That's what the trace is doing... any reason why it wasn't enough?